mlir/test/Dialect/Rock/lowering_xdlops_gemm_v2.mlir

```diff
@@ -1,21 +1,21 @@
 // RUN: rocmlir-opt -rock-threadwise-gemm-lowering %s | FileCheck %s
-#transform_map0 = #rock.transform_map<affine_map<(d0, d1) -> (d0 * 8 + d1)> by [<Embed{8, 1} ["m", "k"] at [0, 1] -> ["raw"] at [0]>] bounds = [1, 8] -> [8]>
-#transform_map1 = #rock.transform_map<affine_map<(d0, d1) -> (d0 * 8 + d1)> by [<Embed{8, 1} ["n", "k"] at [0, 1] -> ["raw"] at [0]>] bounds = [1, 8] -> [8]>
-#transform_map2 = #rock.transform_map<affine_map<(d0, d1, d2) -> (d0 * 2 + d1 * 2 + d2)> by [<Embed{2, 2, 1} ["m", "n", "v"] at [0, 1, 2] -> ["raw"] at [0]>] bounds = [1, 1, 2] -> [2]>
+#transform_map0 = #rock.transform_map<affine_map<(d0, d1) -> (d0 * 2 + d1)> by [<Embed{2, 1} ["m", "k"] at [0, 1] -> ["raw"] at [0]>] bounds = [2, 2] -> [4]>
+#transform_map1 = #rock.transform_map<affine_map<(d0, d1) -> (d0 * 2 + d1)> by [<Embed{2, 1} ["n", "k"] at [0, 1] -> ["raw"] at [0]>] bounds = [1, 2] -> [2]>
+#transform_map2 = #rock.transform_map<affine_map<(d0, d1, d2) -> (d0 * 2 + d1 + d2)> by [<Embed{2, 1, 1} ["m", "n", "v"] at [0, 1, 2] -> ["raw"] at [0]>] bounds = [2, 1, 1] -> [2]>
```

Comment:
Why change this unit test? I don't see any problem with the way it was.

Reply:
The reasoning is similar to the previous reply. We are using reductions now, so for instance in the first example: 
- `A` will be `mRepeats*KPerThreadsxf32` and needs to be transformed in `mRepeatsxKPerThreadxf32`, i.e., `2*2xf32` to `2x2xf32`
- Same reasoning for `B`, but `nRepeats` is one, since `nPerWave=32`
- `C` will be `mRepeats*nRepeats*nvectorsxf32` and need to be transformed to `mRepeats 
 x nRepeats x nvectors x f32`, I.e., from `2*2*1xf32` to `2x2x1xf32`

The same holds for the other example.